Tests/test_Nexus.py

```diff
@@ -647,7 +1136,8 @@ def test_repeated_names_no_taxa(self):
         CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---X
         ;
         end;
-        """)  # noqa : W291
+        """
+        )  # noqa : W291
```

Comment:
Strange. Looks like we can remove this ``# noqa : W291`` and the others still on a closing bracket line?

Reply:
Looks like the only one that's needed is on L730, removed the other two (L1140 and L1170) and flake8 didn't complain. I'll make another commit removing them.